en/docs/release-notes/version-4.5.0.md

```diff
@@ -0,0 +1,30 @@
+# Ballerina VS Code Extension Version 4.5.0
+
+Hello, Ballerina community,
+
+We're excited to present the Ballerina VS Code extension version 4.5.0! Here's what's new:
+
+### What's Added
+
+- **XML Import Option:** Now you can effortlessly create records by importing XML data. Building with XML in Ballerina is more convenient than ever.
+
+    ![Peek 2023-08-22 13-20](https://github.com/kanushka/docs-ballerina-vscode-extension/assets/24244523/eec95f8f-655f-4b7c-9277-a39d4dab5d90)
```

Comment:
These links should be relative to the repo

Reply:
Updated links with this commit https://github.com/wso2/docs-ballerina-vscode-extension/pull/271/commits/dd673c230d4e72ddc5ae18c0e68b70630871b09d